hack/templates/resources/90-shoot.yaml.tpl

```diff
@@ -320,6 +320,12 @@ spec:
   #     SomeKubernetesFeature: true
   #   runtimeConfig:
   #     scheduling.k8s.io/v1alpha1: true
+  #-#-# requires TokenRequest feature gate
+  #-#-# See https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+  #   serviceAccountConfig:
+  #     issuer: "some issuer"
+  #     signingKeySecretName: "my-signing-key-secret"
```

Comment:
Can you also add an example secret that shows end-users how such signing key secret has to look like? We have done the same for the [audit log configmap](https://github.com/gardener/gardener/blob/master/example/95-configmap-custom-audit-policy.yaml), so maybe add it to `example/95-secret-service-account-signing-key.yaml`